index.html

```diff
@@ -3291,6 +3291,23 @@ <h3>Reserved Extension Points</h3>
           </thead>
 
           <tbody>
+            <tr>
+              <td>`confidenceMethod`</td>
+              <td>
+A property used for specifying one or more methods that a verifier 
+might rely on in order to authenticate the holder of a presentation. 
```

Comment:
```suggestion
might use to increase their confidence that the value of an attribute in or of
a verifiable credential or verifiable presentation is accurate, including but not
limited to attributes such `initialRecipient` (a/k/a `issuee`), `presenter`, `authorizedPresenter`, etc.
```

Reply:
Seems odd to remove holder here? 

It's part of the 3 part model, isn't it important to highlight that here?

Reply:
Inadvertent omission in the rewritten sentence. Fixed.